src/index.html

```diff
@@ -11,7 +11,171 @@
+        <legend>Registration</legend>
+        <div class="form_field">
+          <label for="e-mail">E-mail:</label>
+          <input
+            type="email"
+            name="e-mail"
+            required
+            placeholder="email@example.com"
+            id="e-mail"
+          >
+        </div>
+        <div class="form_field">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            required
+            id="password"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form_field">
+          Do you love cats?
+          <label for="Agree">
+            <input type="radio"
+              name="loveCats"
+              value="y"
+              required
+              id="Agree"
+            >
+            Yes
+          </label>
+          <label for="Reject">
+            <input
+              type="radio"
+              name="loveCats"
+              value="n"
+              required
+              id="Reject"
+            >
+            No
+          </label>
+        </div>
+        <div class="form_field">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            multiple
+            id="color"
+          >
+        </div>
+        <div class="form_field">
+          <label for="sleepTime">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="sleepTime"
+            id="sleepTime"
+          >
+        </div>
+        <div class="form_field">
+          <label for="carBrand">What are your favorite brand of cars?</label>
+          <select
+            name="carBrand"
+            id="carBrand"
+            multiple
+          >
+            <option value="Audi">Audi</option>
+            <option value="BMW">BMW</option>
+            <option value="Mercedes">Mercedes</option>
+          </select>
+        </div>
+        <div class="form_field">
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            name="rate"
+            min="0"
+            max="10"
+            id="rate"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldset">
+        <legend>Additional info:</legend>
+        <div class="form_field">
+          <label for="comment">Comments:</label>
+          <textarea
+            name="comment"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+            minlength="0"
+            id="comment"
+          >
+          </textarea>
+        </div>
+        <div class="form_field">
+          <label for="recomandation">Would you recomend us?</label>
+          <select
+            name="recomandation"
+            id="recomandation"
+          >
```

Comment:
Please pay attention to my comments
```suggestion
          <label for="recommendation">Would you recommend us?</label>
          <select
            name="recommendation"
            id="recommendation"
          >
```